tests/unit_tests/test_ladder_service.py

```diff
@@ -52,17 +52,17 @@ async def test_load_from_database(ladder_service, queue_factory):
         assert queue.name == "ladder1v1"
         assert len(queue.map_pools) == 3
         assert list(queue.map_pools[1][0].maps.values()) == [
-            Map(id=15, name="SCMP_015", path="maps/scmp_015.v0003.zip"),
+            Map(id=15, name="SCMP_015", path="maps/scmp_015.v0003.zip", weight=1),
```

Comment:
When you add the default you won't need to change it in the tests

Reply:
yep